backend/cmd/plugins.go

```diff
@@ -135,7 +135,7 @@ func pluginURLsForDir(pluginDir string, baseURL string) ([]string, error) {
 			continue
 		}
 
-		pluginFileURL := filepath.Join(baseURL, f.Name(), "main.js")
+		pluginFileURL := filepath.Join(baseURL, f.Name())
```

Comment:
We should probably also output a warning in this function if the package.js doesn't exist (as we do for main.js).

Reply:
I added a warning if there's no package.json.

We can turn the package.json into a hard requirement in a few versions IMHO. So plugins will keep working at the moment even though there's no package.json. So for now, it returns the plugin if there's no package.json.